### ISSUE TITLE
Task: Build basic funnel for eCommerce trial sidebar nudge

### ISSUE THREAD

**Issue by daledupreez** [Automattic/wp-calypso#74942]
### Details

As part of https://github.com/Automattic/wp-calypso/issues/74548 / D105219-code, we introduced two new Tracks events for the sidebar nudge we show in the left nav, `calypso_wpcom_jitm_ecommerce_trial_impression` and `calypso_wpcom_jitm_ecommerce_trial_cta_click`.

We need to build a basic funnel that tracks the following steps since the nudge was launched:
1. A user saw the nudge - `calypso_wpcom_jitm_ecommerce_trial_impression` (plus maybe some other event props)
2. A user clicked on the nudge - `calypso_wpcom_jitm_ecommerce_trial_cta_click` (plus maybe some other event props)
3. A user clicked on a CTA in the plans page - I can't recall which event we are using for those events!
4. The user purchased a paid eCommerce or Woo Express plan - this should use the `wpcom_product_purchase` event, and will need to OR conditions that require `product_slug` to be either `LIKE(ecommerce-bundle%)` or `LIKE(wp-bundle-wooexpress-%)`

It may be worth doing this as a joint team exercise.

### Checklist

- [x] Build a funnel

### Related

#74548

**Comment by that-mike-bal** [comment]
@daledupreez  is this something we need for or something you wanted to use to validate the JITM fixes OR something you wanted to do for the sake of teaching your team about tracks?

If the events are live I think you can hand it off to another team or even me but let me know.

**Comment by sixhours** [comment]
> The user purchased a paid eCommerce or Woo Express plan - this should use the wpcom_product_purchase event, and will need to OR conditions that require product_slug to be either LIKE(ecommerce-bundle%) or LIKE(wp-bundle-wooexpress-%)

I wasn't having luck segmenting by `product_slug` because I didn't have the right values, so I ran a trial funnel ending with `wpcom_product_purchase` alone to see what product slug(s) to look for and found:

`wooexpress-medium-bundle-monthly`
`wooexpress-medium-bundle-yearly`
`wooexpress-small-bundle-monthly`
`wooexpress-small-bundle-yearly`

I didn't see any `ecommerce-bundle`s in the results, although I did see a couple `business-bundle` purchases.

I re-ran the funnel with `product_slug` => `LIKE(wooexpress-%)` and that pulled back the results I would have expected.

These were my final settings:

Date range March 16th 2023 to April 26 2023, with 7 days to complete the funnel:

1. User sees the JITM in the sidebar - `calypso_wpcom_jitm_ecommerce_trial_impression`
2. User clicks the JITM CTA in the sidebar - `calypso_wpcom_jitm_ecommerce_trial_cta_click`
3. User clicks any upgrade CTA on the Plans page - `calypso_plan_features_upgrade_click` (this doesn't account for the "Plus" plan which would be `calypso_plan_step_woo_express_plus_click`.)
4. User purchases a product with a product slug like `wooexpress-*` or `ecommerce-bundle*` - `wpcom_product_purchase`

I can share the final output in Slack/P2.
